pycqed/instrument_drivers/meta_instrument/LutMans/ro_lutman.py

```diff
@@ -262,105 +324,152 @@ def __init__(self, name, num_res: int=1, **kw):
                            vals=vals.Numbers(min_value=0), unit='s',
                            parameter_class=ManualParameter,
                            initial_value=270e-9)
-
+        self.add_parameter('timeout',
```

Comment:
Is this timeout used? 
It seems like a parameter that belongs in the UHFQA driver, not in the RO Lutman driver.

Reply:
The timeout is transferred to the UHFQA when the RO LutMan initializes it.

I think this highlights a bit of a conceptual problem with the LutMan. The way the LutMan is defined, it basically takes control of the UHFQA and specifies the specific operating mode of the instrument (through the `load_single_pulse_sequence_onto_UHFQC` and `load_DIO_triggered_sequence_onto_UHFQC` methods). These methods were defined with a `timeout` parameter. Therefore, I thought it would make sense to propagate the parameter to the actual instrument where it is used. 

I have the feeling the LutMan is currently in some sort of intermediate state where it manages some of the configuration of the attached UHFQA, but not all.

In my opinion, you should consider to either:
- Not have the RO LutMan configure and control the UHFQA. The user would have to set modes (and also timeout) specifically on the instrument. In this case the `load_single_pulse_sequence_onto_UHFQC` and `load_DIO_triggered_sequence_onto_UHFQC` could be removed from the RO LutMan. However, it would probably break a lot of measurements.
- Accept that the RO LutMan acts as the controller of the UHFQA and then make sure that all necessary initialization functionality is called.